hera_cal/utils.py

```diff
@@ -1513,3 +1827,10 @@ def list_of_int_tuples(v):
                                                                                 "Ex2: '200 300, 500 650' --> [(200, 300), (500, 650), ...]")
     ap.add_argument("--clobber", default=False, action="store_true", help='overwrites existing file at outfile')
     return ap
+
+def mergedicts(*dicts) -> dict:
```

Comment:
This feels unnecessary when you can just do `{k: v for d in dicts for k, v in d.items()}`

Reply:
Fair. I've replaced occurrences of the function with this.